### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by nikolaevtomov**
npm 6.13.4

Screenshot 2020-01-06 at 12 07 33

Required for https://github.com/dabapps/create-webpack-config/pull/17

Updated the config file to fix warnings:

Screenshot 2020-01-06 at 14 23 40

Screenshot 2020-01-06 at 14 09 49

Screenshot 2020-01-06 at 14 09 40

Screenshot 2020-01-06 at 12 07 58

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,4 +3,4 @@ sudo: false
 language: node_js
 
 node_js:
-  - "6.10.2"
+  - "10.5.0"
```
Comment (JakeSidSmith) [inline .travis.yml]: You can, and should, remove the `node_js` config completely. Travis uses the `.nvmrc`.

File: react/.eslintrc.json [hunk 2]
```diff
@@ -6,6 +6,11 @@
   "plugins": [
     "react"
   ],
+  "settings": {
+    "react": {
+      "version": "latest"
```
Comment (JakeSidSmith) [inline react/.eslintrc.json]: Wut?

File: react/.eslintrc.json [hunk 3]
```diff
@@ -16,16 +21,14 @@
...
     "react/jsx-closing-bracket-location": [2, { "nonEmpty": "line-aligned", "selfClosing": "tag-aligned" }],
     "react/jsx-curly-spacing": [2, "never"],
     "react/jsx-equals-spacing": [2, "never"],
     "react/jsx-first-prop-new-line": [2, "multiline"],
     "react/jsx-max-props-per-line": [2, { "maximum": 3 }],
     "react/jsx-pascal-case": [2, { "allowAllCaps": true }],
-    "react/jsx-space-before-closing": [2, "always"],
 
-    "react/jsx-tag-spacing": 2,
+    "react/jsx-tag-spacing": ["error", { "beforeSelfClosing": "always" }],
```
Comment (JakeSidSmith) [inline react/.eslintrc.json]: Why has this changed?

Comment (JakeSidSmith) [reply react/.eslintrc.json]: Also, why not default config?

File: react/.eslintrc.json (hunk 2 above)
Comment (nikolaevtomov) [reply react/.eslintrc.json]: New eslint rules force to specify react version.

Comment (nikolaevtomov) [reply react/.eslintrc.json]: Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .

File: react/.eslintrc.json (hunk 3 above)
Comment (nikolaevtomov) [reply react/.eslintrc.json]: Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .
The react/jsx-space-before-closing rule is deprecated. Please use the react/jsx-tag-spacing rule with the "beforeSelfClosing" option instead.

Comment (nikolaevtomov) [reply react/.eslintrc.json]: The react/jsx-space-before-closing rule is deprecated. Please use the react/jsx-tag-spacing rule with the "beforeSelfClosing" option instead.

Comment (nikolaevtomov) [reply react/.eslintrc.json]: Not sure what we want here.

File: react/.eslintrc.json (hunk 2 above)
Comment (JakeSidSmith) [reply react/.eslintrc.json]: Surely we should use the `detect` option since our projects vary?

File: react/.eslintrc.json (hunk 3 above)
Comment (JakeSidSmith) [reply react/.eslintrc.json]: `"beforeSelfClosing": "always"` is the default. So you can remove the override.

File: react/.eslintrc.json (hunk 2 above)
Comment (nikolaevtomov) [reply react/.eslintrc.json]: `detect` asks to install react. ヾ(´￢｀)ﾉ

File: react/.eslintrc.json (hunk 3 above)
Comment (nikolaevtomov) [reply react/.eslintrc.json]: Removed, ta

File: react/.eslintrc.json (hunk 2 above)
Comment (JakeSidSmith) [reply react/.eslintrc.json]: Add React as a dev dependency.

File: react/.eslintrc.json (hunk 3 above)
Comment (JakeSidSmith) [reply react/.eslintrc.json]: Plz

```suggestion
    "react/jsx-tag-spacing": 2,
```

File: react/.eslintrc.json (hunk 2 above)
Comment (nikolaevtomov) [reply react/.eslintrc.json]: :awyeees:

**Pull request merged**